.vale.ini

```diff
@@ -8,10 +8,6 @@ Vocab = Temporal
 [*]
```

Comment:
Can we change this to 

```
[*.{md,mdx}]
```

It doesn't make sense for it to lint the code files i think. If it's to lint the comments, I don't think we need to apply the same standard as our public docs. This generates a lot of noise for the TS files.